src/main/java/mate/academy/hibernate/relations/Main.java

```diff
@@ -15,18 +21,24 @@ public static void main(String[] args) {
         // use this session factory when you will initialize service instances
         SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
 
+        CountryDaoImpl countryDao = new CountryDaoImpl(sessionFactory);
         Country usa = new Country("USA");
-        CountryService countryService = null; // TODO: initialize this instance
+        CountryService countryService =
+                new CountryServiceImpl(countryDao); // TODO: initialize this instance
         countryService.add(usa);
 
+        ActorDaoImpl actorDao = new ActorDaoImpl(sessionFactory);
```

Comment:
Direct instantiation of `ActorDaoImpl` with `new` should be avoided for the same reasons as with `CountryDaoImpl`.